src/styles/blocks/month.scss

```
@@ -0,0 +1,52 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+body {
+  padding: 0;
+  margin: 50px 0 0;
+  @extend %centre-flex;
+}
+
+.month {
+  display: flex;
+  align-content: flex-start;
+  flex-wrap: wrap;
+  gap: #{$card-gapNborder};
+  width: 750px;
+
+  &__day {
+    @extend %centre-flex;
+    @extend %card-perimeter;
+    background-color: #eee;
+    border: #{$card-gapNborder} solid #000;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: #FFC0CB;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $index from 1 through 31{
+  .month__day:nth-child(#{$index}):before {
+    content: '#{$index}';
+    color: #000;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+}
+
+@each $num, $day in $week-days {
+  .start-#{$day} {
+    margin-left: $num * ($card-square + ($card-gapNborder * 3)) ;
+  }
+}
+
+@for $index from $one-overmonth-length through 31 {
+  .month__day:nth-child(#{$index}) {
+    display: none;
+  }
+}
```

by applying this `for`, you remove all days that should be shown
here is how it works
```suggestion
@for $index from $min-days-num through $max-days-num {
  &--month-length-#{$index} :nth-child(n + #{$index + 1}) {
    display: none;
  }
}
```

also, use nesting, do not place this `for` under everything, you can place it near `month` using `&`